Digital Identity/idpro-account-recovery-final.md

```diff
@@ -370,15 +370,19 @@ Both Knowledge-Based Authentication (KBA) and Security Questions are
 used as recovery mechanisms by having the user “prove” they are the
 legitimate owner by answering questions known only to the user.
 Unfortunately, both KBA, based on public information databases or recent
-user transactions, and security questions, based on preconfigured
+user transactions, previous passwords, and security questions, based on preconfigured
 questions and answers provided by the user, are relatively weak recovery
 mechanisms.
 
 KBA mechanisms often utilize information such as home addresses, loan
 dates/amounts, and credit report data to weakly identify the human owner
 of an account. However, due to numerous data breaches, this information
 is insufficiently secret and should not be depended upon as a recovery
-mechanism for accounts with any significant value. Similarly, security
+mechanism for accounts with any significant value. 
+
+Even in the absense of data breaches, such information is also available to family members which can cause problems when the family is stressed or disolves.
```

Comment:
I like the thought, but I'd change the writing style a bit.  "Information used for KBA may often be available to family members or other parties close to the user, reducing their efficacy."

Reply:
Thanks Dean.  Happy (happier) to use your words.